[terra-table] [terra-data-grid] Add column span functionality to FlowsheetDataGrid and Table (#2096)
cerner/terra-framework: packages/terra-framework-docs/src/terra-dev-site/doc/data-grid/FlowsheetDataGrid.4/Examples.6/FlowsheetDataGridWithColumnSpan.7.doc.mdx

import FlowsheetDataGridWithColumnSpan from './FlowsheetDataGridWithColumnSpan?dev-site-example';

# Flowsheet Data Grid With Column Span

### Description

This example demonstrates a [Flowsheet Data Grid](/components/cerner-terra-framework-docs/data-grid/flowsheet-data-grid/about) with column span support.

<FlowsheetDataGridWithColumnSpan title='Flowsheet Data Grid With Column Span' />

> Note: Column Highlighting functionality will not be supported for columns with column span.

Reviewer: nit: slight wording change since this (i assume) is how it works currently.

```suggestion
Note: Column Highlighting is not supported for columns with column span.
```
